bricks.js

```diff
@@ -1,9 +1,12 @@
 class Ground {
-    constructor(game, x, y, w) {
+    constructor(game, x, y, w, underground) {
         Object.assign(this, { game, x, y, w });
 
         this.spritesheet = ASSET_MANAGER.getAsset("./sprites/bricks.png");
 
+        if (underground) this.spritesheet = ASSET_MANAGER.getAsset("./sprites/SuperMarioBros/sprites/underground_stuff.png");
```

Comment:
This and brick, invalid directory and will crash game upon being drawn (TypeError).